tests/suites/test_suite_ssl.function

```diff
@@ -936,6 +936,54 @@ exit:
     return;
 }
 
+int unhex( char const *hex, unsigned char *dst )
```

Comment:
Which version of unhex is this? Can we use anything in test_suite_helpers.function instead, or add unhex to there?

Reply:
This is the version used in `ssl_client2`. I wanted to switch to a version from the helpers function, but somehow later forgot about that. Will be done.